setup_dartium.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e
+
+DART_DIST="dartsdk-linux-x64-release.zip";
+DARTIUM_DIST="dartium-linux-x64-release.zip";
+SELENIUM_JAR="selenium-server.jar";
+
+echo "Fetching dart sdk and Dartium "
+
+curl "http://storage.googleapis.com/dart-archive/channels/stable/release/latest/sdk/$DART_DIST" > $DART_DIST
```

Comment:
Why does the sdk need to be downloaded again? Isn't it already available on travis? I think I understand dartium since it's used in a symlink below